packages/frontend-2/components/viewer/saved-views/panel/View.vue

```diff
@@ -288,6 +293,16 @@ const wrapperClasses = computed(() => {
     classParts.push('hover:bg-highlight-1')
   }
 
+  if (isViewDragOver.value && dropPosition.value === 'top') {
```

Comment:
should've done this inside the composable, the point of it is to encapsulate all of the dragging logic complexity in there so the Vue component is cleaner/simpler